ace/validator.go

```diff
@@ -359,7 +371,7 @@ func validateAppAnnotations(metadataURL string, pm *schema.PodManifest, app *sch
 			continue
 		}
 
-		val, err := metadataGet(metadataURL, "/apps/"+string(app.Name)+"/annotations/"+key)
+		val, err := metadataGet(metadataURL, "/apps/"+string(app.Name)+"/annotations/"+key, "text/plain; charset=us-ascii")
```

Comment:
@philips I think you wanted to keep parity with aws/gce style metadata services. But if you'd like, we can switch to all JSON. I think in containers people will be doing less bash scripting compared to VM so JSON is probably more appropriate.